tests/anomaly_detection_trade_count.sql

```diff
@@ -1,4 +1,8 @@
-{{ config(severity="warn") }}
+{{ config(
+    severity="warn"
```

Comment:
Just confirming: will the `warn` severity still alert in the slack channel? Otherwise we might have to increase the severity to get the alert in slack

Reply:
Yes, it continues to send alerts even though the severity is configured as warn. This was the case with an alert that was triggering every 30 minutes, and we had to set up a suppression interval to send it only once a day. Here's an example of such an alert for one of these tests: https://indicium-external.slack.com/archives/C06KR1UB6TG/p1716595711802759